Doc/c-api/module.rst

```diff
@@ -484,6 +484,16 @@ state:
 
    Add a string constant to *module*.
 
+.. c:function:: int PyModule_AddType(PyObject *module, PyTypeObject *type)
+
+   Add an type object to *module*. This function calls :c:func:`PyType_Ready`,
+   :c:func:`_PyType_Name()` and :c:func:`PyModule_AddObject()` internally.
```

Comment:
I checked _PyType_Name(): it's not documented. Usually, we avoid mentioning private functions in the public documentation. Instead, the behavior can be document: exactly what you did.

```suggestion
   Add a type object to *module*.
   The type object is finalized by calling internally :c:func:`PyType_Ready`.
```